app/controllers/profiles_controller.rb

```diff
@@ -9,24 +9,30 @@ def index
 
   def show
     # TODO: (Required) Order all these by most prominent first
-    @solutions = @user.solutions.published.first(3)
-    @testimonials = @user.mentor_testimonials.published.first(3)
+    @solutions = @user.solutions.published.first(User::Profile.count_to_show_tab - 1)
+    @testimonials = @user.mentor_testimonials.published.first(User::Profile.count_to_show_tab - 1)
```

Comment:
These should stay as 3 I think. We might change the rules about when the tab shows, but we'd always want to show 3 here regardless.